### PR TITLE
[IMP]10712 add module pos_quantity

### PR DESCRIPTION
**Pull request by Rona111**

### CODE REVIEW

**Comment by elicoidal** [comment]
Please arrange header/README/PEP8

**Comment by Rona111** [comment]
cc @elicoidal @siyuancn @seb-elico @duanyp1991

File: pos_quantity/README.rst [hunk 1]
```diff
@@ -0,0 +1,46 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+======================================
+Order item sequence and total quantity
+======================================
+
+This module extends the functionality of pos to add the order line sequence and total quantity
```
Comment (elicoidal) [inline pos_quantity/README.rst]: s/pos/POS
Add period at end of sentence

File: pos_quantity/static/src/js/quantity.js [hunk 2]
```diff
@@ -0,0 +1,88 @@
+function openerp_pos_quantity(instance, module){
```
Comment (elicoidal) [inline pos_quantity/static/src/js/quantity.js:5]: add copyright

**Comment by elicoidal** [comment]
small details 
LGTM (no technical review)

**Comment by Rona111** [comment]
cc @elicoidal @siyuancn @seb-elico @duanyp1991

**Comment by elicoidal** [comment]
LGTM

**Comment by siyuancn** [comment]
LGTM

**Pull request merged**